conf/layer.conf

```diff
@@ -14,4 +14,6 @@ BBFILE_COLLECTIONS += "prplmesh"
 BBFILE_PATTERN_prplmesh = "^${LAYERDIR}/"
 BBFILE_PRIORITY_prplmesh = "9"
 
-LICENSE_PATH += "${LAYERDIR}/LICENSES"
+LICENSE_PATH += "${LAYERDIR}"
```

Comment:
Why? `LICENSE_PATH` is a list of directories with `LICENSE` files. e.g. when you write `LICENSE = "MIT"` in recipes then yocto search for the file named "MIT" in this directories. 

If you keep your COPYING file on the top level of the layer this does not mean that you should point `LICENSE_PATH` to this directory.

Reply:
Solved in #14